draft-looker-oauth-attested-key-based-client-authentication.md

```diff
@@ -55,20 +55,20 @@ informative:
 
 --- abstract
 
-This specification defines a new method of client authentication for OAuth2 {{RFC6749}} by extending the approach defined in {{RFC7521}}. This new method enables client deployments that are traditionally viewed as public clients to be able to authenticate with the authorization server through an attested key based authentication scheme.
+This specification defines a new method of client authentication for OAuth {{RFC6749}} by extending the approach defined in {{RFC7521}}. This new method enables client deployments that are traditionally viewed as public clients to be able to authenticate with the authorization server through an attestation based authentication scheme.
```

Comment:
does the status of "public clients" be applied also in this case?

I've the perception that in our use cases whatever are the statuses of the clients is not relevant to this draft

Reply:
the clients being authenticated by the approach of this RFC, are to be considered public clients in OAuth speak. Doesn't it make sense to say so?

Reply:
I'm ok with public client since wallet instances are mobile devices, I was just asking since I read in the specs also the role of client backend without knowing which would be their client registration mechanisms